parser/parser.go

```diff
@@ -807,6 +808,16 @@ func (p *parser) primary() ast.Expr {
 			file = p.primary()
 		}
 		return &ast.GetlineExpr{nil, target, file}
+	case PIPE:
+		if p.pendingGetlineLeft == nil {
+			panic(p.errorf("expected expression instead of %s", p.tok))
```

Comment:
Is this possible to hit? What AWK source code would cause it?

Reply:
Two possible (incorrect) programs that trigger the condition are: `BEGIN { | }` and `BEGIN { 1 + 1 - | }`